package gg.bayes.challenge.exception.dto;